app/views/catalog/_show_main_content.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render blacklight_config.track_search_session.item_pagination_component.new(search_context: @search_context, search_session: search_session, current_document: @document) if blacklight_config.track_search_session.item_pagination_component %>
+<%= render Blacklight::Document::PageHeaderComponent.new(document: @document, blacklight_config:, search_context: @search_context, search_session:) unless blacklight_config.full_width_header? %>
```

Comment:
Can we push this class into the config? Can we move the conditional into the component's `render?` method?

Reply:
No because it is shows up in two different places. If it is full_width we want it to render, just somewhere else.

Reply:
But yes we can push the class into config.